src/store2.js

```diff
@@ -105,27 +105,27 @@
                 return !!(this._in(key) in this._area);
             },
             size: function(){ return this.keys().length; },
-            each: function(fn, _and) {// _and is purely for internal use (see keys())
+            each: function(fn, nothing) { // nothing is a default value
                 for (var i=0, m=_.length(this._area); i<m; i++) {
                     var key = this._out(_.key(this._area, i));
                     if (key !== undefined) {
-                        if (fn.call(this, key, _and || this.get(key)) === false) {
+                        if (fn.call(this, key, nothing || this.get(key)) === false) {
                             break;
                         }
                     }
                     if (m > _.length(this._area)) { m--; i--; }// in case of removeItem
                 }
-                return _and || this;
+                return nothing || this;
             },
-            keys: function() {
-                return this.each(function(k, list){ list.push(k); }, []);
+            keys: function(nothing) {
+                return this.each(function(k, list){ list.push(k); }, nothing !== undefined ? nothing : []);
             },
             get: function(key, alt) {
                 var s = _.get(this._area, this._in(key));
                 return s !== null ? _.parse(s) : alt || s;// support alt for easy default mgmt
             },
-            getAll: function() {
-                return this.each(function(k, all){ all[k] = this.get(k); }, {});
+            getAll: function(nothing) {
```

Comment:
I think that:
```function(opts) {
  var nothing = opts && opts.nothing
  // ...
}
```
is better because these code:
```js
.getAll({ nothing: myInitialValue })
```
is self-descriptive